flask_melodramatiq/lazy_broker.py

```diff
@@ -141,10 +146,21 @@ def init_app(self, app):
             self.__configuration = configuration
             options = configuration.copy()
             self.__class__ = options.pop('class')
+
+            # Instanciate dramatiq Broker
             broker = self._dramatiq_broker_factory(**options)
+
+            # Pull out empty backend from middleware
+            if self.__empty_backend in broker.middleware:
+                broker.middleware.remove(self.__empty_backend)
```

Comment:
You instantiate the broker with `self.__empty_backend` (by passing **options), and then remove it. This seems potentially dangerous to me. Dramatiq does not guarantee that this is safe. It seems more robust to not include it in `options` at all. For example, in place of:

```Python
self.__options['middleware'] = self.__stub.middleware
```
you can have something like this:

```Python
self.__options['middleware'] = [m for m in self.__stub.middleware if m is not self.__empty_backend]
```
That is, if I am not missing something.

Reply:
Much better indeed !